internal/stores/contracts.go

```diff
@@ -382,11 +402,11 @@ func (s *SQLStore) AddContract(c rhpv2.Contract) error {
 
 		// Insert contract.
 		return s.db.Where(&dbHost{PublicKey: c.HostKey()}).
-			Create(&dbContractRHPv2{
-				FCID:          fcid,
-				GoodForUpload: true, // new contract is always good for upload
-				HostID:        host.ID,
-				Revision:      revision,
+			Create(&dbContract{
+				FCID:     fcid,
+				IsGood:   true, // new contracts always start as good
```

Comment:
We can change this later if need be but in my head we would keep using `GoodForUpload`, `GoodForRenew` and also introduce an `active` column.

Reply:
Yeah this is just a placeholder. Once the rest of the interface is fleshed out we can modify the query and fields.